src/Pack/Core/Logging.idr

```diff
@@ -105,6 +105,14 @@ export %inline
 info : HasIO io => (ref : LogRef) => (msg  : Lazy String) -> io ()
 info = log ref Info
 
+||| Alias for `log ref Info`.
```

Comment:
This comment is wrong, "`Cache`", not "`Info`"

Reply:
> This comment is wrong, "`Cache`", not "`Info`"

Good catch. I fixed it as part of #232 .